perma_web/requirements.txt

```diff
@@ -87,38 +85,38 @@ Pillow==3.3.1
 pip-tools==1.7
 py==1.4.31                # via pytest
 pyasn1==0.1.9             # via cryptography, requests
-pycparser==2.14           # via cffi
+pycparser==2.16           # via cffi
 pycrypto==2.6.1           # via paramiko
 pyflakes==1.0.0           # via flake8
-pyopenssl==16.0.0         # via certauth, ndg-httpsclient, requests
-pyScss==1.3.4             # via django-pipeline-compass
+pyopenssl==16.2.0         # via certauth, ndg-httpsclient, requests
+pyScss==1.3.4
 pytest-xdist==1.10
-pytest==2.9.1             # via pytest-xdist
+pytest==3.0.3             # via pytest-xdist
 python-dateutil==2.2
-python-mimeparse==1.5.2   # via django-tastypie
+python-mimeparse==1.6.0   # via django-tastypie
```

Comment:
Are these version updates all due to node and webpack integration, or are some of them just updating to latest versions? (e.g. python-mimeparse)